### ISSUE TITLE
Active elements in outline view are black, need more contrast

### ISSUE THREAD

**Issue by kavehv** [guari/eclipse-ui-theme#71]
Because active elements in the [CDT] outline view are black and inactive are grey, it's often hard to see the active elements due to lack of contrast.  It'd be nice if it was possible to fix this.

**Comment by guari** [comment]
Hi @kavehv, a screenshot would help to identify your issue ;)

If I understand it right, currently the appearance of the selected item into a Tree widget is forced by SWT to use the one defined by your OS theme.

**Comment by kavehv** [comment]
![screen shot 2014-07-03 at 8 45 35 am](https://cloud.githubusercontent.com/assets/829620/3472167/21b466bc-02c9-11e4-842e-5f3277fd6402.png)
![screen shot 2014-07-03 at 8 46 35 am](https://cloud.githubusercontent.com/assets/829620/3472174/3d60bdca-02c9-11e4-9ce1-a3bbbd74c316.png)

**Comment by guari** [comment]
Those issues could be solved, are you using Luna Dark theme?

Try if importing the .epf file contained in this repo solves the issue with the Outline view.

For the white ruler in CDT, it's an issue introduced with [Eclipse Bug 435574](https://bugs.eclipse.org/bugs/show_bug.cgi?id=435574) (see [Gerrit review](https://git.eclipse.org/r/#/c/27415/)), check if the current version of this plugin solves it.

**Comment by kavehv** [comment]
I am using Luna Dark (instead of the github sourced Moonrise).  The behavior is inconsistent though.  when I toggle the #if cases in the code, sometimes enabled functions will appear as black, sometimes they won't.  I can try this epf and elaborate more when I get a bigger slice of free time.

**Comment by guari** [comment]
Yes you're right, it seems that the Outline colors for CDT become hard-coded when a snippet of inactive code is written into the code editor. I suggest to open a Bug for it [here](https://bugs.eclipse.org/bugs/buglist.cgi?component=cdt-editor&list_id=9521242&product=CDT&resolution=---).

**Comment by guari** [comment]
Maybe you're the same person that reported it, but [Bug 438485](https://bugs.eclipse.org/bugs/show_bug.cgi?id=438485) should address your issue.
